GliaWidgets/Sources/View/Chat/SecureMessagingBottomBannerViewStyle.RemoteConfig.swift

```diff
@@ -1,3 +1,43 @@
-// TODO: Unified customization will be implemented with MOB-3717
+import UIKit
 
-import Foundation
+extension SecureMessagingBottomBannerViewStyle {
+    mutating func apply(
+        configuration: RemoteConfiguration.SecureConversations?,
+        assetBuilder: RemoteConfiguration.AssetsBuilder
+    ) {
+        configuration?.bottomBannerBackground?.color.unwrap {
+            switch $0.type {
+            case .fill:
+                $0.value
+                    .map { UIColor(hex: $0) }
+                    .first
+                    .unwrap { backgroundColor = .fill(color: $0) }
+            case .gradient:
+                let colors = $0.value.convertToCgColors()
+                backgroundColor = .gradient(colors: colors)
+            }
+        }
+
+        configuration?.topBannerDividerColor?.value
+            .map { UIColor.init(hex: $0) }
+            .first
+            .unwrap {
+                self.dividerColor = $0
+            }
+
+        configuration?.bottomBannerText?.foreground?.value
+            .map { UIColor.init(hex: $0) }
+            .first
+            .unwrap {
+                self.textColor = $0
+            }
+
+        UIFont.convertToFont(
+            uiFont: assetBuilder.fontBuilder(configuration?.bottomBannerText?.font),
+            textStyle: .caption1 // TODO: confirm with reviewers if FontStyle should be hardcoded.
```

Comment:
`textStyle` should be taken from `SecureMessagingBottomBannerViewStyle`

Reply:
Fixed with other suggestions 4fe3105570659aa9e020d60ee26e894a383507cc.